### PR TITLE
Adding parser option for delimiter

### PR DESCRIPTION
**Pull request by kellrott**
Can now parse TSVs with
```
df := dataframe.ReadCSV(r, dataframe.Delimiter('\t'))
```

### CODE REVIEW

File: dataframe/dataframe.go [hunk 1]
```diff
@@ -656,6 +659,13 @@ func WithTypes(coltypes map[string]series.Type) LoadOption {
 	}
 }
 
+// Use Delimiter other then ',', for example '\t'
```
Comment (kniren) [inline dataframe/dataframe.go]: The name of this function should be `WithDelimiter` and the comment should be in the form:

`// WithDelimiter sets the csv delimiter other than ',', for example '\t'`

File: dataframe/dataframe.go [hunk 2]
```diff
@@ -619,6 +619,9 @@ type loadOptions struct {
 
 	// Defines which valeus are going to be considered as NaN when parsing from string
 	nanValues []string
+
+	//csv delimeter
```
Comment (kniren) [inline dataframe/dataframe.go]: `// Defines the csv delimiter`

**Comment by kellrott** [comment]
- Fixed documentation
- Renamed function
- retargeted dev

**Comment by kniren** [comment]
Thank you for your contribution, merged!

**Pull request merged**